lib/msal-common/src/telemetry/server/ServerTelemetryManager.ts

```diff
@@ -49,9 +60,16 @@ export class ServerTelemetryManager {
     /**
      * API to add MSER Telemetry to request
      */
-    generateCurrentRequestHeaderValue(): string {
+    generateCurrentRequestHeaderValue(currentRequest?: string): string {
         const request = `${this.apiId}${SERVER_TELEM_CONSTANTS.VALUE_SEPARATOR}${this.cacheOutcome}`;
-        const platformFields = [this.wrapperSKU, this.wrapperVer].join(
+        const platformFieldsArr = [this.wrapperSKU, this.wrapperVer];
+        if (currentRequest?.length) {
```

Comment:
Instead of passing this data in through request properties can we follow the model of `wrapperSKU` and `wrapperVer`?

Reply:
I wish I could. The goal here is to propagate a native broker error and append it to the last group of `x-client-current-telemetry` parameter so I can't make this value static.

Reply:
Doesn't need to be static - I'd suggest adding a function to the ServerTelemetryManager to set the error when you get one and then read from that when building the header value